### PR TITLE
First draft for Landscape: Update term "Repositories" entry #50

### PR DESCRIPTION
**Pull request by TropicolX**
Providing first draft of the `repositories.md` in path: `landscape/explanation/glossary/repositories.md` #50

### CODE REVIEW

File: landscape/explanation/glossary/repositories.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Repositories
+
+**Repositories** are the foundation for software distribution in Linux environments. They hold all the necessary files and resources needed for projects. Linux distributions like Ubuntu maintain repositories containing thousands of pre-built packages you can install on your managed computers.
```
Comment (YanisaHS) [inline landscape/explanation/glossary/repositories.md:3]:
```suggestion
**Repositories** are the foundation for software distribution in Linux environments. They hold all the necessary files and resources needed for projects. Linux distributions like Ubuntu maintain package repositories containing thousands of pre-built packages you can install on your managed computers.
```

Comment (YanisaHS) [reply landscape/explanation/glossary/repositories.md:3]: I'm suggesting this change to make it more clear that the repos Landscape deals with are _package_ repositories (i.e., not source-code repositories).

**Review by YanisaHS — CHANGES_REQUESTED** [review]
This looks really good! I just have that one comment for you to address :smile:

File: landscape/explanation/glossary/repositories.md (hunk 1 above)
Comment (TropicolX) [reply landscape/explanation/glossary/repositories.md:3]: Yeah, that makes sense!

**Review by TropicolX — COMMENTED** [review]
Looks good!

**Comment by YanisaHS** [comment]
@TropicolX Thank you so much for your wonderful contribution!! :partying_face: :balloon: :tada: Your contribution is now live here: https://ubuntu.com/landscape/docs/explanation-repositories

Looking forward to working with you again! :smiley:

**Comment by TropicolX** [comment]
Awesome! I'm thrilled to see it live! 🌟 Thanks for the opportunity, and I can't wait for our next project together

**Pull request merged**